# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome24/gtkextra/fix_autogen.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./autogen.sh.old	Thu Jul 18 20:55:51 2002
+++ ./autogen.sh	Sun Feb  2 22:41:49 2003
@@ -50,7 +50,8 @@
 if ( aclocal --version ) </dev/null > /dev/null 2>&1; then
 	rm -f aclocal.m4
 	echo -n "Building aclocal.m4... "
-	${ACLOCAL}
+	${ACLOCAL} -I /opt/gnome2/share/aclocal
+	${ACLOCAL} -I /opt/gnome24/share/aclocal
 	echo "done."
 else
 	echo "aclocal not found -- aborting `basename $0`"
